src/Http/Controllers/VoyagerDatabaseController.php

```diff
@@ -135,6 +136,24 @@ public function update(Request $request)
             );
     }
 
+    public function cleanOldAndCreateNew($originalName, $tableName)
```

Comment:
remove the entry from permissions & data_types tables, create a model for the new name